hws/src/dr_common.py

```diff
@@ -152,10 +152,11 @@ def hex_to_bin_str(_n, _len):
 STC_ACTION_JUMP_TO_VPORT = '85'
 
 
-def hit_addr_calc(next_table_base_39_32, next_table_base_31_5):
-    hit_addr = next_table_base_39_32 << 32
+def hit_addr_calc(next_table_base_63_48, next_table_base_39_32, next_table_base_31_5):
+    hit_addr = next_table_base_63_48 << 40
+    hit_addr |= next_table_base_39_32 << 32
     hit_addr |= (next_table_base_31_5 << 5)
-    hit_addr = (hit_addr >> 6) & 0xffffffff
+    hit_addr = hit_addr >> 6
```

Comment:
why take the whole address, this will break the tool flow

Reply:
> see dr_hw_get_ste_from_addr

This still works fine for me. Admittedly I'm dumping the eswitch domain and the GVMI is 0, so perhaps that's hiding some issues right now.

If there are any issues here, we need to fix those too. Perhaps the addresses/keys of _db._fw_ste_db need to include the GVMI ID too. Do they not already? If we fixed that, then we could add DB entries for the FW-pref-defined STEs that e.g. implement DROP into the database, which would be very useful...

Reply:
Per our conversation, I'll push an updated version of the pull request that works correctly for GVMI != 0.

Reply:
I've updated the code to handle this.

... although the pre-existing issue index values more than 32-bits (address values more than 38 bits) still exists.